src/main/java/uk/gov/hmcts/ccd/domain/service/casedeletion/CaseLinkMapper.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.ccd.domain.service.casedeletion;
+
+import uk.gov.hmcts.ccd.data.caseaccess.CaseLinkEntity;
+import uk.gov.hmcts.ccd.domain.model.casedeletion.CaseLink;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.inject.Named;
+
+@Named
+public class CaseLinkMapper {
+
+    public CaseLink entityToModel(final CaseLinkEntity caseLinkEntity) {
```

Comment:
Can mark parameter as `@NonNull` to avoid the null check

Reply:
Done [c129a9a](https://github.com/hmcts/ccd-data-store-api/pull/1872/commits/c129a9a6a6fdff4226e65e0b027691d3cd49352f)